### PR TITLE
Givermei checklistload

### PR DESCRIPTION
**Pull request by vermegi**
The feature itself is finished, however, the checklist does not load properly, because of a bug further down the line in changeTemplate (Ft3asApp.tsx line 50). There is something wrong with the parsing, seems to me ...

### CODE REVIEW

**Comment by vermegi** [comment]
As in: the english checklists are loading all fine, it's the ones in another language that give a problem on load ...

File: src/controls/Ft3asTemplateSelector.tsx [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+
+    const [selectedTechItem, setSelectedTechItem] = useState<IDropdownOption>();
+    const [selectedLanguageItem, setSelectedLanguageItem] = useState<IDropdownOption>();
+
     const onOk = () => {
-        props.onTemplateSelected(selectedItem?.key as string | undefined);
+        if (selectedLanguageItem && selectedTechItem){
+            console.log(selectedTechItem.text);
+            console.log(selectedLanguageItem.text);
+            var url : string = TemplateServiceInstance.getPathforTechAndLanguage(selectedTechItem.text, selectedLanguageItem.text);
```
Comment (felipmiguel) [inline src/controls/Ft3asTemplateSelector.tsx:42]: I would replace _var url_ by _const url_ or _let url_ if it is mutable. The behavior of var in javascript is very unpredictable as it behaves as a global variable.

**Review by felipmiguel — CHANGES_REQUESTED** [review]
Looks good. There is only one variable declaration using _var_ that could lead to unexpected issues. An sample explanation https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference

File: src/controls/Ft3asTemplateSelector.tsx (hunk 1 above)
Comment (vermegi) [reply src/controls/Ft3asTemplateSelector.tsx]: fixed.

**Pull request merged**